packages/dev/core/src/Materials/Node/nodeMaterial.ts

```diff
@@ -72,8 +72,10 @@ declare let BABYLON: any;
  * Interface used to configure the node material editor
  */
 export interface INodeMaterialEditorOptions {
-    /** Define the URl to load node editor script */
+    /** Define the URL to load node editor script from */
     editorURL?: string;
+    /** If the scene's clear color should be used as the NME preview's clear color */
+    useSceneClearColor?: boolean;

```

Comment:
Would it be more flexible to have a color parameter instead ?

PPL could use it to pass any colors and internally we would pass the scene.clearColor, what do you think ?

Reply:
Valid, I just made a property to pass any of the NME config options 😃